### DISCUSSION TITLE
Get Client Files Method

### DISCUSSION THREAD

**Discussion by oksuzgil** [mynaparrot/plugNmeet-server#305] [General]
Example server domain: a.com
Example client domain: b.com

Hello, css and js files are coming in the getClientFiles method. But config.js is not coming. The page can be displayed when Config.js is added. Does it need to be added to the api?

When the page opens, the system should load the language files from a.com, but it is searching under b.com. It cannot be displayed because it cannot be found.

The information "You just need to add the <div id="plugNmeet-app"></div> tag to the page you want to add should be added to the document of the getClientFiles method.

**Comment by jibon57** [comment]
Please check example from plugins. You'll have to create all config manually. The URL should be always plugNmeet url.

**Comment by oksuzgil** [comment]
Hello, I got it. We will add config.js constant. Is it possible to add language files in the system to the same method? In this way, we can import the relevant file according to the language of the user.{
    "css": [
        "vendor.xxx.css",
        "main.xxx.css"
    ],
    "js": [
        "main.xxx.js",
        "runtime.xxx.js",
        "tflite-simd.js",
        "tflite.js",
        "vendor.xxx.js"
    ],
   "locales": [
     en,
     tr-TR,
...
    ],
    "msg": "success",
    "status": true
}
